pygmt/tests/test_plot.py

```diff
@@ -231,6 +242,42 @@ def test_plot_colors_sizes_proj(data, region):
     return fig
 
 
+@check_figures_equal()
```

Comment:
@core-man Please rewrite the test following our new workflow. See https://github.com/GenericMappingTools/pygmt/issues/963#issuecomment-801606095 for details.

Reply:
@seisman could you please add me as a collaborator (https://dagshub.com/GenericMappingTools/pygmt/settings/collaboration) as said in [CONTRIBUTING.md](https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#using-data-version-control-dvc-to-manage-test-images)? See my DAGsHub account (https://dagshub.com/core-man).

Reply:
Done.

Reply:
done in 3b24385.